ddsrouter_yaml/include/ddsrouter_yaml/validator/ddsrouter_config_schema.json

```diff
@@ -0,0 +1,477 @@
+{
+    "$schema":"http://json-schema.org/draft-07/schema#",
+    "$ref":"#/definitions/BaseObject",
+    "definitions":{
+        "BaseObject":{
+            "type":"object",
+            "additionalProperties":false,
+            "properties":{
+                "version":{
+                    "type":"string",
+                    "enum":[
+                        "2.0"
```

Comment:
TODO: Check this with the expected tags for the version.
I would say to call them `v2.0` instead of `2.0`. The second option could lead to parse it as a float instead of string when parsing to json